samples/bluetooth/peripheral_gap_svc/src/main.c

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2025 Koppel Electronic
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/gatt.h>
+
+
+#define DEVICE_NAME		CONFIG_BT_DEVICE_NAME
+#define DEVICE_NAME_LEN		(sizeof(DEVICE_NAME) - 1)
+
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+	BT_DATA(BT_DATA_NAME_COMPLETE, DEVICE_NAME, DEVICE_NAME_LEN),
+};
+
+/* -----------------------------------------------------------------------------
+ * Local implementation of GAP service
+ */
+
+static ssize_t read_name(struct bt_conn *conn, const struct bt_gatt_attr *attr,
+			 void *buf, uint16_t len, uint16_t offset)
+{
+	const char *name = bt_get_name();
+
+	return bt_gatt_attr_read(conn, attr, buf, len, offset, name,
+				 strlen(name));
```

Comment:
Minor: We use a max width of 100 characters in Zephyr, so this could be


```suggestion
	return bt_gatt_attr_read(conn, attr, buf, len, offset, name, strlen(name));
```